sdk/deviceupdate/Azure.IoT.DeviceUpdate/CHANGELOG.md

```diff
@@ -1,10 +1,4 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
-
-
-## 1.0.0-beta.2 (2021-04-06)
-* Update root namespace from Azure.Iot.DeviceUpdate to Azure.IoT.DeviceUpdate
-
-## 1.0.0-beta.1 (2021-03-03)
+## 1.0.0-beta.1 (2022-01-07)
```

Comment:
Why were entries for previous releases removed?  You cannot re-release a package to nuget with the same name of an existing library, and it looks like 1.0.0-beta.1 has already been released.